examples/simple_counter/Android/shared/build.gradle

```diff
@@ -83,16 +83,16 @@ afterEvaluate {
 }
 
 tasks.register('bindGen', Exec) {
-    def outDir = "${projectDir}/../../shared_types/generated/java"
+    def outDir = "${projectDir}/../../shared/generated/java"
     workingDir "../../"
     if (System.getProperty('os.name').toLowerCase().contains('windows')) {
         commandLine("cmd", "/c",
-                "cargo build -p shared && " + "target\\debug\\uniffi-bindgen generate shared\\src\\shared.udl " + "--language kotlin " + "--out-dir " + outDir.replace('/', '\\'))
+                "cargo build -p shared && " + "target\\debug\\uniffi-bindgen generate --library target\\debug\\libshared.dylib " + "--language kotlin " + "--out-dir " + outDir.replace('/', '\\'))
```

Comment:
I may have misunderstood what the additional lines in cargo config do -  I thought they add helper commands for this step, which we could then use in build scripts like this? Is that not the case?

Reply:
Yes, nice catch. We could utilize the alias config here too.